src/main/java/org/folio/service/transformationfields/TransformationFieldsConfig.java

```diff
@@ -8,16 +8,36 @@
  * Initial data for the transformation field. While extending the enum, put new values in alphabetical order
  */
 public enum TransformationFieldsConfig {
-
+  //Common Fields
   ID("id", "$.{recordType}.id"),
   HR_ID("hrid", "$.{recordType}.hrid"),
-  INSTANCE_ID("instanceId", "$.holdings.instanceId"),
   IDENTIFIERS("identifiers", "$.{recordType}[*].identifiers[?(@identifierTypeId=={id})].value", IDENTIFIER_TYPES),
-
+  SOURCE("source", "$.{recordType}.source"),
   METADATA_CREATED_DATE("metadata.createdDate", "$.{recordType}.metadata.createdDate", MetadataParametersConstants.getFixedLengthDataElement()),
   METADATA_UPDATED_DATE("metadata.updatedDate", "$.{recordType}.metadata.updatedDate"),
   METADATA_CREATED_BY_USER_ID("metadata.createdByUserId", "$.{recordType}.metadata.createdByUserId"),
-  METADATA_UPDATED_BY_USER_ID("metadata.updatedByUserId", "$.{recordType}.metadata.updatedByUserId");
+  METADATA_UPDATED_BY_USER_ID("metadata.updatedByUserId", "$.{recordType}.metadata.updatedByUserId"),
+
+  //Instance specific fields
+  EDITIONS("editions", "$.instance.editions"),
```

Comment:
I understand that is specific for this type of record, but maybe it is better to use "$.{recordType}.editions" for jsonPath,

Reply:
Is there a reason? I left the one's that will be common among multiple types with recordType.  
The advantage in this way we will be able to easily keep track of which fields are required for each Record type

Reply:
ok, it is not a big deal